desktop/src/main/java/bisq/desktop/main/offer/OfferViewUtil.java

```diff
@@ -166,4 +172,13 @@ public static Stream<CryptoCurrency> getMainCryptoCurrencies() {
         return CurrencyUtil.getMainCryptoCurrencies().stream().filter(cryptoCurrency ->
                 !Objects.equals(cryptoCurrency.getCode(), GUIUtil.TOP_ALTCOIN.getCode()));
     }
+
+    public static void submitTransactionHex(XmrWalletService xmrWalletService,
+                                             TableView tableView,
+                                             String reserveTxHex) {
+        MoneroSubmitTxResult result = xmrWalletService.getDaemon().submitTxHex(reserveTxHex);
+        tableView.refresh();
+        new Popup().information(Res.get("support.info.submitTxHex", result.toString())).show();
+        //TODO Display any errors
```

Comment:
This todo should be resolved. As is, this popup is displayed to the arbitrator:

<img width="689" alt="image" src="https://user-images.githubusercontent.com/13068859/216159267-5b5a609c-6842-4a4d-9a85-e75e9e549f48.png">

They should get an indication of error on failure to publish.